test/spec/controllers/createPatientRequestMapper.spec.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+describe('CreatePatientRequestMapper', function () {
+
```

Comment:
@valvijo Yes, I got confuse because of the module dependencies and controllers is testing.
Will also add  more content for the Testing document to make the Guidelines clear.